addons/delivery/views/delivery_view.xml

```diff
@@ -12,7 +12,7 @@
                     <field name="sequence" widget="handle"/>
                     <field name="name"/>
                     <field name="partner_id"/>
-                    <field name="price" invisible="'order_id' not in context"/>
+                    <field name="price"/>
```

Comment:
remove price field form the list view, add delivery type to be display in the list

use name_get method to generate the name like 
Normal Delivery ($ 10.00)
Express Delivery ($ 25.00)